pkg/generator_test.go

```diff
@@ -702,6 +707,127 @@ func (s *GeneratorSuite) TestGeneratorVariadicArgs() {
 	)
 }
 
+func (s *GeneratorSuite) TestGeneratorVariadicArgsAsOneArg() {
+	// Read the expected output from a "golden" file that we can also import in CompatSuite
+	// to asserts its actual behavior.
+	//
+	// To regenerate the golden file: make fixture
+	expectedBytes, err := ioutil.ReadFile(filepath.Join(fixturePath, "mocks", "requester_variadic_one_arg.go"))
+	s.NoError(err)
+	expected := string(expectedBytes)
+	expected = expected[strings.Index(expected, "// RequesterVariadicOneArgument is"):]
+	generator := NewGenerator(
```

Comment:
I created a new generator manually again as it needs new `Vararg` flag.
Does it make sense to make `checkGeneration` accept config? maybe in a different PR, or does it better suit mockery `v3`?

Reply:
I think making a new generator is the right way to do it. Generator objects should be tied to static config. I have no problems with this.